djangosaml2/views.py

```diff
@@ -378,6 +378,14 @@ def logout(request, config_loader_path=None):
 
     result = client.global_logout(subject_id)
 
+    try:
+        result = client.global_logout(subject_id)
```

Comment:
Looks like a duplicate of line 379 above. Maybe that line 379 needs to be deleted?

Reply:
You're right, I'll update this branch with the code i've in pplnx-dev

Reply:
Here https://github.com/peppelinux/djangosaml2/commit/734a87e803d2e3f41ce08e3026c9de38ee92dce7